enterprise/backend/src/metabase_enterprise/enhancements/integrations/ldap.clj

```diff
@@ -73,7 +74,7 @@
                           (ldap-group-membership-filter))]
       (assoc user-info :attributes (syncable-user-attributes result)))))
 
-(defenterprise-schema fetch-or-create-user! :- (class User)
+(defenterprise-schema fetch-or-create-user! :- (mi/InstanceOf User)
```

Comment:
`(class User)` is `clojure.lang.Keyword` in Toucan 2, but I added `mi/InstanceOf` a while back which is compatible with either Toucan 1 or Toucan 2. I switched most things over to this already, but I guess I missed this spot.